test/safe-hts-precompile/SafeHTS.js

```diff
@@ -320,10 +320,8 @@ describe('SafeHTS library Test Suite', function () {
     )
 
     const signers0AfterTokenBalance = await pollForNewBalance(erc20Mock, signer1AccountID, signers0BeforeTokenBalance)
-  
-    const signers1AfterTokenBalance = parseInt(
-      await erc20Mock.balanceOf(signer2AccountID)
-    )
+
+    const signers1AfterTokenBalance = await pollForNewBalance(erc20Mock, signer2AccountID, signers1BeforeTokenBalance)
```

Comment:
`signers1AfterTokenBalance` is never used. Wha was it added for?

Reply:
`signers1AfterTokenBalance` is used on line 344 in an `expect` validation:
`    expect(signers1AfterTokenBalance).to.equal(
      signers1BeforeTokenBalance + transferredAmount
    )
`

Reply:
Read the git change as a new variable. Seems it existed and was used already